packages/system/src/Item/api/hooks/useItemsList/useItemsList.ts

```diff
@@ -8,11 +7,14 @@ import {
   UseQueryResult,
   useQueryClient,
 } from '@openmsupply-client/common';
-import { getItemSortField, mapItemNodes } from '../../utils';
+import { useItemApi } from './../useItemApi';
```

Comment:
Changes to use the code generated `ItemFragment` rather than the `Item` type and also use the co-located generated operations query rather than from common